attelo/parser/intra.py

```diff
@@ -30,6 +30,56 @@ class IntraInterPair(namedtuple("IntraInterPair",
     pass
 
 
+def for_intra(dpack, target):
+    '''
```

Comment:
Yeah, that's worth fixing.  Should really be spotted automatically.  Never trust a man to do a machine's job.